test_unstructured_ingest/unit/pipeline/reformat/test_chunking.py

```diff
@@ -0,0 +1,144 @@
+from __future__ import annotations
+
+import os
+import tempfile
+
+import pytest
+from _pytest.logging import LogCaptureFixture
+
+from test_unstructured.unit_utils import (
+    FixtureRequest,
+    Mock,
+    example_doc_path,
+    function_mock,
+    method_mock,
+)
+from unstructured.documents.elements import CompositeElement
+from unstructured.ingest.interfaces import ChunkingConfig, PartitionConfig
+from unstructured.ingest.pipeline.interfaces import PipelineContext
+from unstructured.ingest.pipeline.reformat.chunking import Chunker
+
+ELEMENTS_JSON_FILE = example_doc_path(
+    "test_evaluate_files/unstructured_output/Bank Good Credit Loan.pptx.json"
+)
+
+
+class DescribeChunker:
+    """Unit tests for ingest.pipeline.reformat.chunking.Chunker"""
+
+    # -- Chunker.run() -----------------------------------------------------------------------------
+
+    # -- integration test --
+    def it_creates_json(self, _ingest_docs_map_: Mock):
+        chunking_config = ChunkingConfig(chunking_strategy="by_title")
+        pipeline_context = PipelineContext()
+        partition_config = PartitionConfig()
+        chunker = Chunker(
+            chunking_config=chunking_config,
+            pipeline_context=pipeline_context,
+            partition_config=partition_config,
+        )
+        with tempfile.TemporaryDirectory() as tmpdir:
+            # -- `Chunker.chunk()` defaults to writing to "{work_dir}/chunked", which is located in
+            # -- "/.cache" of a user's profile.
+            # -- Define `work_dir` add the "/chunked" subdirectory to it:
+            chunker.pipeline_context.work_dir = tmpdir
+            os.makedirs(os.path.join(tmpdir, "chunked"), exist_ok=True)
+
+            filename = chunker.run(ELEMENTS_JSON_FILE)
+            head, tail = os.path.split(filename if filename else "")
+            # -- Check that a json file was created in `/chunked` --
+            assert head.endswith("chunked")
+            assert tail.endswith(".json")
+
+    def it_logs_error_with_invalid_remote_chunking_args(
```

Comment:
The name says invalid remote chunking args but the test only tests invalid chunking strategy. Those two should agree. I'm thinking change the name.

It also makes me wonder how it knows the chunking strategy is invalid but I'll read on.

Reply:
How about "it_logs_error_on_invalid_remote_chunking_strategy()`.